tests/programs/dlopen_demo.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Run the shared library dynamic loading demo program.
+# This is only expected to work when Mbed TLS is built as a shared library.
+
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+
+. "${0%/*}../../scripts/demo_common.sh"
```

Comment:
```suggestion
. "${0%/*}/../../scripts/demo_common.sh"
```
With that fix the CI of #9939 should be green. Otherwise the framework pointer in #9988 does not seem to be correct.